### DISCUSSION TITLE
Necessity of implementing Mertens algorithm for percolation

### DISCUSSION THREAD

**Discussion by harveywilliams56** [Part-II-Computational-Physics/.github#96] [Q&A]
Hi, I'm currently working on the percolation project and wonder if it is sufficient to implement just the Newman-Ziff algorithm with perhaps discussion but not implemention of other algorithms. Will this be penalised heavily/are we expected to implement both?

The Mertens algorithm itself is fairly straight-forward but getting useful results from it is much more mathematically involved than for the Newman-Ziff algorithm and would add a substantial amount of work.

**Comment by AustenLamacraft** [comment]
It’s not necessary to implement Mertens’ algo but it would be good to have
some discussion of the other (types of) approaches e.g transfer matrix vs
Monte Carlo.
